### ISSUE TITLE
Update plugin

### ISSUE THREAD

**Issue by soyaforcemeat** [iekmuby/woocommerce-loyalty-cards#1]
Hi!  
Are you planning to update your plugin?

**Comment by iekmuby** [comment]
Hi. What exactly you want to update? This plugin currently in progress (change structure to OOP, some improvements).

**Comment by soyaforcemeat** [comment]
![image](https://user-images.githubusercontent.com/24391080/101287492-3a558680-37f9-11eb-8db8-56da31403d4e.png)
I thought it's due new wc wp versions. How i see your last commit was 3 years ago

**Comment by iekmuby** [comment]
You're right. This plugin is in progress right now, but I can't provide exact dates when I finish it.

**Comment by soyaforcemeat** [comment]
Ok! Thank you!
Your plugin would be very helpful for my new project.

**Comment by soyaforcemeat** [comment]
Unfortunately, I can not find anything similar to your plugin.
